src/stylesheets/components/top-fold.scss

```diff
@@ -0,0 +1,133 @@
+/*-------- Top Fold styles ---------*/
+
+.top-fold {
+  background: $florence-light-blue;
+  overflow: hidden;
+  padding-top: 50px;
+
+  .title-box {
+    position: relative;
+  }
+
+  .title {
+    margin: auto;
+    max-width: 50vw;
+
+    @media (max-width: $screen-sm) {
+      margin-bottom: 30px;
+      max-width: 85vw;
+    }
+  }
+
+  .subtitle {
+    color: $florence-blue-green;
+    font-family: $tertiary-font;
+    font-size: 14px;
+    letter-spacing: 3px;
+    margin-bottom: 30px;
+    text-align: center;
+    text-transform: uppercase;
+
+    @media (max-width: $screen-sm) {
+      margin-bottom: 15px;
+    }
+  }
+
+  .yellow-pill {
+    left: 22%;
+    position: absolute;
+    top: -30%;
+    transform: rotate(35deg);
+    width: 5em;
+
+    @media (max-width: $screen-sm) {
+      top: -55%;
+      left: 1%;
+      transform: rotate(40deg);
+    }
+  }
+
+  .pills {
+    left: 12%;
+    position: absolute;
+    top: 65%;
+    width: 8em;
+
+    @media (max-width: $screen-sm) {
+      display: none;
+    }
+  }
+
+  .more-pills {
+    position: absolute;
+    right: 12%;
+    top: -20%;
+    transform: rotate(10deg);
+    width: 15em;
+
+    @media (max-width: $screen-sm) {
+      right: -35%;
+      top: -40%;
+    }
+  }
+
+  .purple-pills {
+    position: absolute;
+    right: 10%;
+    width: 9em;
+
+    @media (max-width: $screen-sm) {
+      bottom: -30%;
+      right: -22%;
+      z-index: 4;
+    }
+  }
+}
+
+/*----------- Pillbar styles ----------*/
+
+.pillbar {
+  background-image: linear-gradient(0deg, $florence-red 75%, $florence-blue 25%);
+  background-image: -webkit-linear-gradient(0deg, $florence-red 75%, $florence-blue 25%);
```

Comment:
How about we go ahead and move the red/blue backgrounds to the `<li>` elements in this PR, and then move the border-radius to the left and right elements respectively. Also, you could use the `:first-child` and `:last-child` selectors for this instead of using your `left-end` and `right-end` classes.

Reply:
Looks like there might be more opportunity for using pseudo-classes instead of classes - e.g. `:nth-child(…)` for the left and right factor boxes, and maybe the other instances of ordinal classes